share/trick/makefiles/trickify.mk

```diff
@@ -86,15 +86,29 @@
 # -----------------------------------------------------------------------------
 #
 # For more information, see:
-# github.com/nasa/trick/wiki/Trickified-Project-Libraries
+# https://nasa.github.io/trick/documentation/building_a_simulation/Trickified-Project-Libraries
 
 ifndef TRICKIFY_CXX_FLAGS
     $(error TRICKIFY_CXX_FLAGS must be set)
 endif
 
 TRICKIFY_BUILD_TYPE ?= PLO
 TRICKIFY_OBJECT_NAME ?= trickified.o
+
+# We started zipping the Python modules, and the original variable is now misnamed.
+# For backward compatibility, initialize the new variable from the old variable if set, and then
+# use the old vairable for something else! python.zip would be a better default value, but leaving
+# it as python will allow existing Trickified projects that don't override the default to work both
+# before and after this change.
 TRICKIFY_PYTHON_DIR ?= python
+# Use origin instead of ?=, which defaults to recursive expansion, since we change
+# TRICKIFY_PYTHON_DIR after this assignment and don't want that affecting TRICKIFY_PTYHON_ZIP.
```

Comment:
I've read `PYTHON` and `PTYHON` so many times now, I don't know which is correct!